### PR TITLE
Spoilers command

### PR DESCRIPTION
**Pull request by Shadran**
Adds the spoiler command to moebot, which converts the given text to a GIF that will be shown after 3 seconds.

Usage: prefix spoiler [Spoiler Title] Spoiler Text

### CODE REVIEW

File: moebot_bot/util/gifmaker.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	"image/color"
+	"image/gif"
+	"strings"
+
+	"golang.org/x/image/font"
+	"golang.org/x/image/font/basicfont"
+	"golang.org/x/image/math/fixed"
+)
+
+const lineSpace int = 16
```
Comment (camd67) [inline moebot_bot/util/gifmaker.go]: Was `go fmt` run on these files? I'm not sure if it catches these, but if we've got a block of constants we should put them into a `const()` block
```
const (
    lineSpace int = 16
    xBorder int = 30
    yBorder int = 14
    maxWidth int = 500
)
```

File: moebot_bot/util/gifmaker.go [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+const xBorder int = 30
+const yBorder int = 14
+const maxWidth int = 500
+
+func MakeGif(text string) []byte {
+	const defaultText string = "Hover to view"
```
Comment (camd67) [inline moebot_bot/util/gifmaker.go]: This and the consts above don't need the `string` and `int`. Go will infer this from the right side (unless you don't want the default types). I'd say it'd be good to remove these types

**Comment by camd67** [comment]
Besides those two style points, looks good to me 👍 I'll accept once they're fixed up

File: moebot_bot/util/gifmaker.go (hunk 1 above)
Comment (Shadran) [reply moebot_bot/util/gifmaker.go]: I ran goreturns (which should in turn call goimports and gofmt)  on the file but it didn't pick up those problems.  
I made sure to fix them manually.

**Pull request merged**